functions/src/modules/pm/categories/prizes/updatePrize.ts

```diff
@@ -1,5 +1,58 @@
-import { ExpressFunction } from "../../../../@types";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
 
-export default replaceMe;
+import type { ExpressFunction } from "../../../../@types";
+import { firebaseApp } from "../../../../config/firebaseConfig";
+import {
+  expressErrorHandlerFactory,
+  requestBodyTypeValidator,
+} from "../../../../helpers";
+import { uasPermissionSwitch } from "../../../../systems/uas";
+
+
+const updatePrize: ExpressFunction = (req, res, next) => {
+  uasPermissionSwitch({
+    organizer: { accepted: ValidateOrganizer },
```

Comment:
This should be camelCase.
```suggestion
    organizer: { accepted: validateOrganizer },
```